sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -425,8 +427,9 @@ class RecognizePiiEntitiesResult(DictMixin):
         ~azure.ai.textanalytics.TextDocumentStatistics
     :ivar bool is_error: Boolean check for error item when iterating over list of
         results. Always False for an instance of a RecognizePiiEntitiesResult.
+
     .. versionadded:: v3.1
-        The *redacted_text* parameter.
+        The *redacted_text* property.
```

Comment:
hmm so pii was only added in 3.1 are you seeing that for 3.0 pii was there and in 3.1 redacted_text was added?

Reply:
Let me check - but this model shouldn't have been there so yes, this should be removed.